### PR TITLE
fix: update 5.0 post with breaking change

### PR DESCRIPTION
**Pull request by davidxia**
`--incompatible_restrict_string_escapes` was removed
[here](https://github.com/bazelbuild/bazel/commit/21b4b1a489c991682b9fcc9a10b468b9ea18e931)

cc @Wyverald 

see https://github.com/bazelbuild/bazel/pull/14695

### CODE REVIEW

**Comment by Wyverald** [comment]
@aiuto thoughts?

**Review by hicksjoseph — APPROVED** [review]
These seems like a pretty straightforward change.

**Comment by aiuto** [comment]
I think we have a process question here.
Why is anyone but a single release manager for all of 5.x reviewing anything related to 5.x?

**Comment by Wyverald** [comment]
This is the blog. I mean I am fine with just approving and merging this blog post, but normally any blog post changes get reviewed by one of you guys, no?

If you're talking about the PRs against the 5.1 branch in bazelbuild/bazel, then yes, the release manager should be reviewing and merging them (as I have been). Sometimes Github automatically assigns reviewers due to CODEOWNERS, but there's not much we can do there.

**Comment by aiuto** [comment]
A product manager should approve all blog posts. For now, that is Joe Hicks.
I didn't phrase myself well. It was more a question of why our approval process allows anyone else a reviewer.
We should clean up the codeowners.

**Pull request merged**
